### DISCUSSION TITLE
How can I remove a table's top search bar? (searchable() is not present)

### DISCUSSION THREAD

**Discussion by RX-77IT** [filamentphp/filament#3773] [Help] [answered]
Hi all! Bit of a newbie here.

Yesterday I did a composer update and I discovered that all my tables and relation managers now have a non working top search bar.

No column has the ->searchable() method, so... how do I remove the search bar when I don't need it?

**Comment by danharrin** [comment, marked as answer]
Sorry, I've pushed a fix for this, it'll be available in the next release

**Reply by RX-77IT** [reply]
Thank you!
